src/main/kotlin/calculator/Operator.kt

```diff
@@ -0,0 +1,25 @@
+package calculator
+
+import java.util.function.BiFunction
+
+enum class Operator(
+    private val symbol: String,
+    private val function: BiFunction<Int, Int, Int>
```

Comment:
```suggestion
    private val function: (Int, Int) -> Int
```
람다를 사용하게 되면, 좀 더 코틀린스럽게 작성할 수 있어요.